jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -91,6 +91,12 @@
       <unit id="org.slf4j.api" version="1.7.2.v20121108-1250"/>
     </location>
 
+    <location includeAllPlatforms="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/tern/0.8.0.201501132311-SNAPSHOT/"/>
+      <!-- Required by FeedHenry module -->
```

Comment:
required by feedhenry AND tern I assume. any reason why we are getting this from requirements/tern and not some more uniform place.

otherwise we got requirements/tern -> TP -> requirements/tern -> JSDT which just seems weird.